bundle/manifests/enterprise.splunk.com_clustermasters.yaml

```diff
@@ -3802,6 +3842,66 @@ spec:
     storage: true
     subresources:
       status: {}
+  - name: v1
```

Comment:
The bundle updates were generated after running `make bundle`. I'm validating if these should or not be part of the PR. 
CC: @vivekr-splunk , @sgontla